main_logic/tts_client.py

```
@@ -990,3 +1007,241 @@ def get_tts_worker(core_api_type='qwen', has_custom_voice=False):
+            # 1. 获取请求 (非阻塞获取以便处理连接状态)
+            try:
+                # 在 loop 中运行 request_queue.get 以便支持打断
+                loop = asyncio.get_running_loop()
+                # ==== 调试点 2 正在等待队列 ===
+                'TODO:清理测试的代码'
+                print(">>> [DEBUG] 正在等待 request_queue.get ...")
+                #
+                sid, tts_text = await loop.run_in_executor(None, request_queue.get)
+                # === 调试点 3：获取到了数据 ===
+                print(f">>> [DEBUG] 队列收到数据! sid={sid}, text={tts_text}")
+            except Exception:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)